src/app/body/my-dashboard/my-tasks/my-tasks.component.ts

```diff
@@ -16,29 +18,25 @@ export class MyTasksComponent implements OnInit {
 
   tasksCollection: AngularFirestoreCollectionGroup<Tasks>
   tasksData: Observable<TasksId[]>
+  orgDomain: string
 
-  constructor(public db: AngularFirestore) { }
+  constructor(public db: AngularFirestore, public backendService: BackendService,private functions: AngularFireFunctions) { }
 
   ngOnInit(): void {
+    this.orgDomain = this.backendService.getOrganizationDomain();
     console.log("1");
     this.readTaskData();
   }
 
-  readTaskData() {
-    console.log(this.currentSprintNumber);
-    this.tasksCollection = this.db.collectionGroup<Tasks>("Tasks", ref => {
-      let queryRef = ref;
-      queryRef = queryRef.where('SprintNumber', '==', this.currentSprintNumber);
-      queryRef = queryRef.where('Assignee', '==', this.userEmail);
-      return queryRef;
-    });
-    this.tasksData = this.tasksCollection.snapshotChanges().pipe(
-      map(actions => actions.map(a => {
-        const data = a.payload.doc.data() as Tasks;
-        const id = a.payload.doc.id;
-        console.log(data);
-        return { id, ...data };
-      }))
-    );
-  }
+  async readTaskData() {
+    const callable = this.functions.httpsCallable('tasks');
+    try{
+      const result = await callable({mode: "getTasksCard",orgDomain:this.orgDomain, sprintNumber:this.currentSprintNumber, Assignee: this.userEmail}).toPromise();
+      this.tasksData = result.taskData;
```

Comment:
Better to have the observable so that you don't need to do changes in html file. Check with komal on this.